colavo/lib/email/notification-service.ts

```
@@ -0,0 +1,381 @@
+  static async getEventNotifications(eventId: string): Promise<typeof scheduledNotifications.$inferSelect[]> {
+    return await db
+      .select()
+      .from(scheduledNotifications)
+      .where(
+        and(
+          eq(scheduledNotifications.entityId, eventId),
+          eq(scheduledNotifications.type, 'event')
+        )
+      );
+  }
+} 
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)